StartBonusActivationOrchestrator/handler.ts

```diff
@@ -35,7 +34,7 @@ export const handler = function*(
     return false;
   }
   const errorOrBonusVacanzaBase = toApiBonusVacanzaBase(
-    errorOrStartBonusActivationOrchestratorInput.value.bonusActivationRequest
+    errorOrStartBonusActivationOrchestratorInput.value.bonusActivation
```

Comment:
I suggest to assign 

`const startBonusActivationOrchestratorInput = errorOrStartBonusActivationOrchestratorInput.value;`

and use that one